### PR TITLE
Module 7: Continuous Integration

### PR DESCRIPTION
**Pull request by CharlieCumber**

### CODE REVIEW

**Comment by Jonesey13** [comment]
Designated Reviewer: Alex

**Review by Jonesey13 — CHANGES_REQUESTED** [review]
.

File: README.md [hunk 1]
```diff
@@ -68,11 +68,26 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
...
+The end to end tests require you have the Chrome browser installed and that you download the corresponding version of the ChromeDriver:
+* Download and install Chrome from [`here`](https://www.google.co.uk/chrome/)
+* Download ChromeDriver from [`here`](https://sites.google.com/a/chromium.org/chromedriver/downloads) and add the executable file to the root of the project.
 
 Once the test setup requirments are completed, execute the tests by running:
 ```bash
 $ poetry run pytest
 ```
+
+### Option 2: Docker
```
Comment (JackMeadDev) [inline README.md:81]: Nice clear instructions

**Review by JackMeadDev — APPROVED** [review]
Nice, some really clear changes here and I can see that travis built & ran your tests successfully against this pull request

**Pull request merged**